pyproject.toml

```diff
@@ -13,16 +13,14 @@ license = "LGPL-3.0"
 
 [tool.poetry.dependencies]
 python = "^3.6"
-django = {version = "^2.2", extras = ["django"]}
+django = {version = "^2.2", extra = ["django"]}
```

Comment:
Actually, I think `extras` is the right word. And it should be `extras` for Flask too.

Reply:
At least that's what the example here suggests: https://github.com/sdispater/poetry#introduction

Reply:
`poetry install` fails with `extras`.

Reply:
yeah that's cos it's right =) what we should actually be doing is `optional = True` or something I think

Reply:
https://poetry.eustace.io/docs/pyproject/#extras

Reply:
But the simplest thing to do would just be to purely update the version in these files for the time being. None of this stuff is related to this PR.

Reply:
Yup, but that then fails for `poetry-setup` :man_shrugging:

Reply:
But FWIW, I think the right syntax should be:

``` toml
django = {version = "^2.2", optional = true}
flask = {version = "^1.0", optional = true}

[tool.poetry.extras]
django = ["django
flask = ["flask"]
```

Reply:
Yeah but you don't *need* to fix or run `poetry-setup` for this PR do you? You can literally just update the number in `setup.py` and `pyproject.toml`

Reply:
Yeah, done that now.